lib/bgs/student_school.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module BGS
+  class StudentSchool
+    def initialize(proc_id:, vnp_participant_id:, payload:, user:)
+      @user = user
+      @proc_id = proc_id
+      @vnp_participant_id = vnp_participant_id
+      @dependents_application = payload['dependents_application']
+    end
+
+    def create
+      bgs_service.create_child_school(submit_params(child_school))
+      bgs_service.create_child_student(submit_params(child_student))
+    end
+
+    private
+
+    def child_school
+      @child_student = BGSDependents::ChildSchool.new(
```

Comment:
The `@child_student` instance variable gets overwritten in `def child_student` - is that intentional? or maybe this variable needs to be renamed to `@child_school`?

Reply:
Also, why does this need to exist as an instance variable at all?

Reply:
Also also, `def child_school` & `def child_student` look pretty thin. Why not delete these private methods and just add inline instantiations in `def create`? i.e.

```ruby
child_school = BGSDependents::ChildSchool.new(@dependents_application)
child_student = BGSDependents::ChildStudent.new(@dependents_application)
```

Reply:
Thanks for the recommendation, I've updated as suggested.